documentation/src/content/docs/en/essentials/events.md

```diff
@@ -0,0 +1,283 @@
+
+```ts
+import { createEvent, sample } from "effector";
+
+const firstTriggered = createEvent<void>();
+const secondTriggered = createEvent<void>();
+
+sample({
+  clock: firstTriggered,
+  target: secondTriggered,
+});
+```
+
+When the `firstTriggered` event is invoked, the `secondTriggered` event will be subsequently called, creating a sequence of events.
+[Remember, dont call events in pure functions](/en/introduction/core-concepts#purity), it's not supported!
+
+:::tip{title="Good to know"}
+In Effector, any event supports only **a single argument**.
+It is not possible to call an event with two or more arguments, as in `someEvent(first, second)`.
+:::
+
+All arguments beyond the first will be ignored.
+The core team has implemented this rule for specific reasons related to the design and functionality.
+This approach enables the argument to be accessed in any situation without adding types complexity.
+
+If multiple arguments need to be passed, encapsulate them within an object:
+
+```ts
+import { createEvent } from "effector";
+
+const requestReceived = createEvent<{ id: number; title: string }>();
+
+requestReceived({ id: 1, title: "example" });
+```
+
+This rule also contributes to the clarity of each argument's meaning, both at the call side and subscription side. It promotes clean and organized code, making it easier to understand and maintain.
+
+## Watching the event (#event-watch)
+
+To ascertain when an event is called, effector and its ecosystem offer various methods with distinct capabilities. Debugging is the primary use case for this purpose, and we highly recommend using [`patronum/debug`](https://patronum.effector.dev/methods/debug/) to display when an event is triggered and the argument it carries.
+
+```ts
+import { createEvent, sample } from "effector";
+import { debug } from "patronum";
+
+const firstTriggered = createEvent<void>();
+const secondTriggered = createEvent<void>();
+
+sample({
+  clock: firstTriggered,
+  target: secondTriggered,
+});
+
+debug(firstTriggered, secondTriggered);
+
+firstTriggered();
+// => [event] firstTriggered undefined
+// => [event] secondTriggered undefined
+```
+
+However, if your environment does not permit the addition of further dependencies, you may use the `watch` method with caution.
+
+```ts
+import { createEvent, sample } from "effector";
+
+const firstTriggered = createEvent<void>();
+const secondTriggered = createEvent<void>();
+
+sample({
+  clock: firstTriggered,
+  target: secondTriggered,
+});
+
+firstTriggered.watch(() => console.info("[event] firstTriggered"));
+secondTriggered.watch(() => console.info("[event] secondTriggered"));
+
+firstTriggered();
+// => [event] firstTriggered
+// => [event] secondTriggered
+```
+
+:::tip{title="Keep in mind"}
+The `watch` method neither handles nor reports exceptions, manages the completion of asynchronous operations, nor addresses data race issues.
+
+Its primary intended use is for short-term debugging and logging purposes.
```

Comment:
Also it's valid to use `createWatch` in tests to ensure that some unit was triggered

Reply:
i think it's better to mention in testing page, what do you think ?